### DISCUSSION TITLE
' make gui_final ' command doesn't work

### DISCUSSION THREAD

**Discussion by ReemAbuRajab7** [The-OpenROAD-Project/OpenROAD-flow-scripts#2508] [Q&A]
why ' make gui_final ' can't work without ' DESIGN_CONFIG=path  ' ? 
I think its related to some dependency, but which one ? 
or it's another problem ? 
![aask](https://github.com/user-attachments/assets/bb0173a8-b2b1-4837-8e9a-e7fb13dd0354)
![aask](https://github.com/user-attachments/assets/0e4827ba-e62b-44ce-b75a-d1d66b42a1a8)

**Comment by maliberty** [comment]
It needs to know which design to load and that is controlled by DESIGN_CONFIG.

**Reply by ReemAbuRajab7** [reply]
![2](https://github.com/user-attachments/assets/a9bdf130-5aeb-4af5-8768-1c7aea0b0846)
![1](https://github.com/user-attachments/assets/990b6e22-7e71-475c-bf3e-8f1c790bbba5)

**Reply by maliberty** [reply]
You have a good eye for detail.  The image is a bit old.  The specific number of instances will change over time.  You can find the clock tree on the windows menu.  The messages are printed to the terminal before the GUI is opened now.  Pin markers were moved under `Shape Types`.  The design name was added to the title bar.

You could make a PR with an updated image if you wish.
